src/chapters/05/main.tex

```diff
@@ -415,10 +418,10 @@ \section{Solving with R}\label{sec:solving-with-R}
 }
 \end{Rin}
 
-Using this function returns the instantaneous rate of change for each of the
-three stocks, $S$, $I$ and $R$. If we begin at time 0.0, with 4 susceptible
+This function returns the instantaneous rate of change for each of the
+three stocks, $S$, $I$ and $R$. Starting at time 0.0, with 4 susceptible
```

Comment:
Sotcks? Quantities? Functions? We should stick to one.

Reply:
I like quantities.

Reply:
:+1: I'll go through and change them all to quantiles. I might in the theory section say they are also sometimes called stocks (I use a stock and flow diagram), and that they are essentially functions